docs/development/extensions-contrib/prometheus.md

```diff
@@ -29,7 +29,16 @@ To use this Apache Druid extension, [include](../../development/extensions.md#lo
 
 This extension exposes [Druid metrics](https://druid.apache.org/docs/latest/operations/metrics.html) for collection by a Prometheus server (https://prometheus.io/).
 
-Emitter is enabled by setting `druid.emitter=prometheus` [configs](https://druid.apache.org/docs/latest/configuration/index.html#emitting-metrics) or include `prometheus` in the composing emitter list. 
+Emitter is enabled by setting `druid.emitter=prometheus` [configs](https://druid.apache.org/docs/latest/configuration/index.html#enabling-metrics) or include `prometheus` in the composing emitter list. 
+
+In certain instances, Druid processes may be colocated on the same host. For example, the Broker and Router may share the same server. Other colocated processes include the Historical and MiddleManager or the Coordinator and Overlord. When you have colocated processes, specify `druid.emitter.prometheus.port` separately for each process on each host. For example, even if the Broker and Router share the same host, the Broker runtime properties and the Router runtime properties each need to list `druid.emitter.prometheus.port`, and the port value for both must be different.
+
+Peon tasks are short-lived batch processes typically colocated with MiddleManagers and often Historicals. Multiple peons may run at once. These tasks should use `pushgateway` for `druid.emitter.prometheus.strategy` and should not start an HTTP server to push metrics. 
```

Comment:
There's a section below the configuration to describe the configuration for peon tasks. 
I think this paragraph is not necessary.